### DISCUSSION TITLE
Hi, how to revoke the windows API

### DISCUSSION THREAD

**Discussion by liaodalin19903** [quasarframework/quasar#11177] [CLI - Electron mode]
I want to develop a windows electron application with  quasar, 

and in it I will revoke [desktop-dup-api](https://docs.microsoft.com/en-us/windows/win32/direct3ddxgi/desktop-dup-api), 
I want to know is it possible to do it?
and what I should learn?

**Comment by hawkeye64** [comment]
This is a tough one. Looks like the API is in C#, so you'll need to know that. Then learn how to convert your C# output (library) to WASM. Learn how to add WASM into the Node build. Then how to access the WASM from your JavaScript.
Seems a bit tedious, but I don't know any other way this could be done unless you find a JavaScript library that has interface bindings to the desktop-dup-api already.
